README.md

```diff
@@ -1,87 +1,55 @@
 # OpenShift Lightspeed operator
 
-A Kubernetes operator for managing OpenShift Lightspeed service.
+A Kubernetes operator for managing [Red Hat OpenShift Lightspeed](https://github.com/openshift/lightspeed-service).
 
 ## Getting Started
 
-You'll need a OpenShift cluster to run against.
+You'll need an OpenShift cluster to run against.
 
-**Note:** Your controller will automatically use the current context in your kubeconfig file (i.e. whatever cluster `oc cluster-info` shows).
+> [!IMPORTANT]
```

Comment:
shall we use `**IMPORTANT**` here to make it bold?
This does not render special format in my markdown preview ^^" 
![image](https://github.com/openshift/lightspeed-operator/assets/6040960/ff2ee618-c0ec-4a26-a547-70d5e84a6442)

Reply:
It's [GH-specific syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)

https://github.com/openshift/lightspeed-operator/blob/a5f041141a351457655d2b5e7db1a75a0b2d80fa/README.md